### DISCUSSION TITLE
Vercel Deployment Website

### DISCUSSION THREAD

**Discussion by DBStock1** [vercel/community#6452] [Help] [answered]
### Question

404: NOT_FOUND
Code: NOT_FOUND
ID: bom1::xfl8h-1712586884252-20092cffbf74 . I am getting this error from two days for website deployment . Kindly solve the problem.

**Comment by amyegan** [comment]
Hey @DBStock1. We need more information about your project. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that we can use to see what's going wrong?

**Comment by DBStock1** [comment]
I am trying to deploy the attach file in vercel . But getting error .



With regards,
Rakesh Lahkar

**Comment by DBStock1** [comment]
Kindly check Amy

**Comment by DBStock1** [comment]
What type of information do you need further ? Please let me know. It's a
HTML file that I am trying to host using Vercel Platform, I have tried
other platforms like Netlify. It is working but I have domain name issues
so I preferred Vercel . Anyway if it's not possible in Vercel not an issue
I just wanted to know the ERROR reason .

**Reply by swarnava** [reply]
We need a GitHub repo that can be deployed on Vercel or tested locally.

**Comment by DBStock1** [comment]
https://github.com/DBStock1/zapakk

**Reply by amyegan** [reply, marked as answer]
If you rename `Zapakk.html` to `index.html`, then that page will load as the home page for your site. Then you should no longer see the `404: NOT_FOUND` error.

**Comment by DBStock1** [comment]
Thanks, it worked.
